styles/index.scss

```diff
@@ -24,9 +21,8 @@ html {
 }
 
 body {
-  font: 300 getFontSize(0) 'Open Sans', 'Century Gothic', sans-serif;
+  font: 300 getFontSize(0) 'Cabin', 'Century Gothic', sans-serif;
```

Comment:
I'd suggest using 400 for body text not 300. < 400 gets a bit hard to read on many devices.